File: backend/common/src/main/java/com/ociet/loadgenerator/common/LoadResultMessage.java
package com.ociet.loadgenerator.common;

import lombok.*;

@ToString
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class LoadResultMessage {
    private String requestKey;
    private long responseTimeSum;
    private long maxResponseTime;
    private long startTimestamp;
    private boolean failed;
}

File: backend/common/src/main/kotlin/com/chylb/loadgenerator/common/LoadResultMessage.kt
package com.chylb.loadgenerator.common

class LoadResultMessage(
    val requestKey: String? = null,
    val responseTimeSum: Long = 0,
    val maxResponseTime: Long = 0,
    val startTimestamp: Long = 0,
    val error: String? = null,
)
{}